### PR TITLE
chore: [IOPID-938] Opt-in screen

### PR DESCRIPTION
**Pull request by Ladirico**
## Short description
By implementing the Fast Login there is a need to make the user choose whether to use the standard login or to use the Quick Login. I implemented this screen by referring to [this mockup ](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?type=design&node-id=2843-27901&mode=design&t=1rt63XBjxBPMAyTB-0)

## List of changes 
proposed in this pull request
- Screen realization NewOptInScreen.tsx
- Test screen NewOptInScreen.test.tsx
- Insertion of navigation logic in LandingScreen.tsx
- Integration of business logic 

https://github.com/pagopa/io-app/assets/83651704/4d0101ae-d3fc-499e-858e-531726f34869

## How to test
In **.env file**  `FAST_LOGIN_ENABLED=YES` and `FAST_LOGIN_OPTIN=YES`
in io-dev-server-api in backend.ts file:

```ts
fastLogin: {
  min_app_version: {
    ios: "1.2.1",
    android: "0.0.0"
  },
  opt_in: {
    min_app_version: {
      ios: "1.2.1", 
      android: "0.0.0"
    }
  }
}
```
NB: if fastLogin is disabled (`0.0.0`) the opt-in is disabled too, but not the reverse. 
Run the application

### CODE REVIEW

**Comment by Ladirico** [comment]
Hi @AliceAzzolini !
would you please check the English texts?
Thanks!

**Review by AliceAzzolini — APPROVED** [review]
I've edited the file myself and made a new commit, so it's all good to go now.

File: ts/navigation/params/AuthenticationParamsList.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
+import { ChoosedIdentifier } from "../../screens/authentication/NewOptInScreen";
 import { CieCardReaderScreenNavigationParams } from "../../screens/authentication/cie/CieCardReaderScreen";
 import { CieConsentDataUsageScreenNavigationParams } from "../../screens/authentication/cie/CieConsentDataUsageScreen";
 import { CieWrongCiePinScreenNavigationParams } from "../../screens/authentication/cie/CieWrongCiePinScreen";
 import ROUTES from "../routes";
 
 export type AuthenticationParamsList = {
   [ROUTES.AUTHENTICATION_LANDING]: undefined;
+  [ROUTES.AUTHENTICATION_OPT_IN]: ChoosedIdentifier;
```
Comment (shadowsheep1) [inline ts/navigation/params/AuthenticationParamsList.ts:9]:
```suggestion
  [ROUTES.AUTHENTICATION_OPT_IN]: ChoosenIdentifier;
```

File: ts/screens/authentication/NewOptInScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+import { useIODispatch } from "../../store/hooks";
+import { TranslationKeys } from "../../../locales/locales";
+
+// FIXME -> insert correct contextual help and FAQ https://pagopa.atlassian.net/browse/IOPID-987
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "authentication.landing.contextualHelpTitle",
+  body: "authentication.landing.contextualHelpContent"
+};
+
+export type ChoosedIdentifier = {
```
Comment (shadowsheep1) [inline ts/screens/authentication/NewOptInScreen.tsx:37]:
```suggestion
export type ChoosenIdentifier = {
```

File: ts/navigation/params/AuthenticationParamsList.ts [hunk 3]
```diff
@@ -1,10 +1,12 @@
+import { ChoosedIdentifier } from "../../screens/authentication/NewOptInScreen";
```
Comment (shadowsheep1) [inline ts/navigation/params/AuthenticationParamsList.ts:1]:
```suggestion
import { ChoosenIdentifier } from "../../screens/authentication/NewOptInScreen";
```

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: ts/navigation/params/AuthenticationParamsList.ts (hunk 1 above)
Comment (Ladirico) [reply ts/navigation/params/AuthenticationParamsList.ts]: done

File: ts/navigation/params/AuthenticationParamsList.ts (hunk 3 above)
Comment (Ladirico) [reply ts/navigation/params/AuthenticationParamsList.ts]: done

**Review by shadowsheep1 — APPROVED** [review]
LGTM 👍

**Pull request merged**